pkg/build/builder/daemonless.go

```diff
@@ -306,10 +306,76 @@ func buildDaemonlessImage(sc types.SystemContext, store storage.Store, isolation
 func generateTransientMounts() []string {
 	mounts := []string{}
 	mounts = appendRHSMMount(mounts)
+	mounts = appendETCPKIMount(mounts)
+	mounts = appendRHRepoMount(mounts)
 	mounts = appendCATrustMount(mounts)
 	return mounts
 }
 
+func appendRHRepoMount(mounts []string) []string {
+	st, err := os.Stat("/run/secrets/redhat.repo")
+	if err != nil {
+		// since the presence of /run/secrets/redhat.repo is not a given, we won't log this a V(0) like
+		// we do for /run/secrets/rhsm
+		log.V(4).Infof("Failed to stat /run/secrets/redhat.repo, falling back to the Red Hat yum repository configuration in the build's base image. Error: %v", err)
```

Comment:
re: your comment https://github.com/openshift/builder/pull/228#issuecomment-805110570 @adambkaplan
this should stay a V4 ... this more likely to happen than not, so we only need to know if we want tracing to confirm